### ISSUE TITLE
White Screen on Launch (new 3DS XL)

### ISSUE THREAD

**Issue by EpicNicks** [DS-Homebrew/TWiLightMenu#1196]
**Your console configuration:**  
New 3DS firmware 11.9.0; Luma3DS v. 10.0.1
**TWiLight Menu++ Version:** 
16.3.0 Release
**TWiLight Menu++ Theme:** 
DSi (I guess)

### Expected behaviour

Twilight Menu++ should start the opening animations and enter the actual menu. 

### Actual behaviour

The top and bottom screen remain white indefinitely. (This issue used to happen sometimes and now happens always)

### Steps to reproduce

1. Start TwilightMenu++
2. Waited 30 minutes with no signs of starting

**Comment by RocketRobz** [comment]
Try deleting `settings.ini` at `sd:/_nds/TWiLightMenu/`

**Comment by EpicNicks** [comment]
That's the first thing I tried. Unfortunately didn't work.

**Comment by EpicNicks** [comment]
Removed all original files from SD and reinstalled the updater.
Installed the TwilightMenu++ (Release) then nds-bootstrap (Release).
Installed Extras (BoxArt).
Entered Settings (TwilightMenu++ did not appear after install), TwilightMenu++ appeared on homescreen.
Launched TwilightMenu++.
Launched in about 10-15 seconds.

Was probably some corruption or something.

**Comment by albertofustinoni** [comment]
I'm having the exact same issue, but no way to fix so far

**Comment by RocketRobz** [comment]
@albertofustinoni Have you tried reformatting the SD card to FAT32 with 32kb cluster size.

**Comment by albertofustinoni** [comment]
My SD card is already formatted with a 32kb cluster size. I know because I did that a few hours ago...

Now, the SD card in question is a 256GB one, if that has any bearing.

**Comment by RocketRobz** [comment]
Nope, but apparently for large SD cards, it is.

**Comment by HAMBRUGNATCHERS** [comment]
I think it has to do with nightly bootstrap. I changed mine from nightly to release and it no longer happens. At least not yet. During mid animation of the logo press select and it’ll take you to settings. Third from the bottom of games and apps is bootstrap. Change it to release. I have no idea what bootstrap does and I would like to know, but if the problem returns I’ll be sure to let you know.

**Comment by RocketRobz** [comment]
The bootstrap option changes the build of nds-bootstrap.
Said option does not cause this issue.

**Comment by HAMBRUGNATCHERS** [comment]
I feel like an idiot and I know this isn’t the place for this question. **First** thanks for getting back so quickly. **Second** I have no idea what nds-bootstrap means, is, or what it does. I would love to know. **Third** I would like to boot “Pokémon: Black Version” in DSi mode. Every time I try I get this. 
![EBC16DFF-0391-490E-A4D2-05C26E4E47BF](https://user-images.githubusercontent.com/48148239/95112573-1ac5b300-070f-11eb-91a6-b0b52651c437.jpeg)

**Comment by RocketRobz** [comment]
1. No problem.
2. nds-bootstrap is what runs the DS ROMs from the console's SD card.
3. Just do what's stated in the message, and wait for DSi mode to be implemented into nds-bootstrap.

**Comment by HAMBRUGNATCHERS** [comment]
Awesome thank you. Do you know of a good source for the rom. I have a physical copy of Pokémon: Black Version 1, can I use duo a rom from it. I also want Pokémon: Black Version 2, which I do not have a physical copy for that one, is there a reliable website to get one that can boot in DSi mode?

**Comment by RocketRobz** [comment]
Yes, you can dump Black 1.
As for Black 2, you'll have to find the ROM yourself, as we don't assist with piracy.

Also, there's [this](https://discord.gg/yD3spjv) Discord server.

**Comment by chyyran** [comment]
Closing in favour of #1211

**Comment by HAMBRUGNATCHERS** [comment]
I found the fix for this. All you have to do is close and open the screen. I don’t know why this works, but PokeChest pops up a message saying: If loading progress freezes, close and reopen the DS. I wondered if that would also fix the issue with the startup screen of TWiLight Menu, so I tried it. I’ve done multiple attempts and it has worked every time so far. Your welcome.

**Comment by popy2k14** [comment]
Same issue with white screen sometimes on 2x different DSIs-XL.
See my description here: https://github.com/DS-Homebrew/TWiLightMenu/issues/1211#issuecomment-746166009

**Comment by FinTriesCode** [comment]
Im having the following issue related to the static white screen on any .nds or TWiLight Menu++ launch:
---------------------------------------------------------------------------------------------------------

I have injected homebrew onto the SD card and followed instructions from there.

It runs .cias files ok

The issue is adding backwards compatibility (built in ds games). I have gotten as far as adding them to the device and having them show on the home screen however, when clicked on, it just loads a white screen and then gets suck (the white screen that usually progressed to the "nintendo ds" bootstrap/intro screen). sometimes i can go to home screen but nothing happens on reboot or relaunch.

Twilight Menu++ does the same thing.

I have all (that Im aware of) software required:
TwiLight Menu++
NDS Bootleg
NDS Forwarder

Still all white screen on boot.

I have tried a reformatting of the SD card, reinstall of all mentioned apps, readding new dsl files and purging files followed by another reinstall.

Been at it for hours lol
